spec/jobs/job_configurations_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'job_configurations' do
+  describe 'weekly report' do
+    %w[drop_off_report authentication_report].each do |job_name|
+      it "schedules the #{job_name.humanize} to run after the end of the week with yesterday's date" do
```

Comment:
I would not bother with `job_name.humanize` because it doesn't tie in to anything we can cmd-f to search for
```suggestion
      it "schedules #{job_name} to run after the end of the week with yesterday's date" do
```